pkg/templates/templates_generated.go

```diff
@@ -1352,6 +1360,18 @@ downloadCNI() {
     retrycmd_get_tarball 120 5 "$CNI_DOWNLOADS_DIR/${CNI_TGZ_TMP}" ${CNI_PLUGINS_URL} || exit $ERR_CNI_DOWNLOAD_TIMEOUT
 }
 
+{{- if IsKrustlet }}
+downloadKrustlet() {
+    mkdir -p $KRUSTLET_DOWNLOAD_DIR
```

Comment:
```suggestion
    if [ -f "$KRUSTLET_BIN_DIR/krustlet-wasi" ]; then
         echo "Krustlet already installed, no op"
         return
    fi
    mkdir -p $KRUSTLET_DOWNLOAD_DIR
```

Reply:
this allows for vanilla ubuntu to download from storage while vhd will use prebaked binaries (best practice is keeping behavior comparable)